rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form-original.json

```diff
@@ -558,13 +544,26 @@
         "openmrs_entity_id": "",
         "type": "edit_text",
         "hint": "Body temperature ℃",
+        "edit_type": "number",
+        "v_numeric": {
```

Comment:
I don't think this has to be a rounded number.

Reply:
it's removed.